package.json

```diff
@@ -69,5 +69,9 @@
     "react-dom": "^16.14.0",
     "react-trello": "^2.2.11",
     "string-to-color": "^2.2.2"
+  },
```

Comment:
I guess that `resolutions` not works on npm.
Ref. https://docs.npmjs.com/cli/v7/configuring-npm/package-json

In this sample, `react` and `react-dom` are direct-dependency.
How about you to stable these versions in L68-L69?

```
    "react": "16.14.0",
    "react-dom": "16.14.0",
```

Reply:
@chick-p 
resolutions is a property used by yarn.
[Ref](https://yarnpkg.com/configuration/manifest#resolutions)

In fact, using npm to compile the project will not have any compatibility issues.

However, using yarn to compile the project will load react18 and cause an error.

 ```
    "react": "16.14.0",
     "react-dom": "16.14.0",
```
Using this method, yarn will fail during the build phase.

Therefore, I personally think that using resolutions is a better solution

Reply:
@cyaoc 
Yes. `resolutions` is a yarn option.
Even if you stable dependency version, 
you mean that the sub-dependency version may not be resolved.

In README.md, introduce to use npm.
If users have to use yarn, I want you to rewrite it.

Reply:
@chick-p 
Yes, you are right.
I have added the instructions for yarn in the readme